tsconfig.json

```diff
@@ -1,14 +1,64 @@
 {
     "compilerOptions": {
-        "target": "es5",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */
-        "allowJs": true,                       /* Allow JavaScript files to be compiled. */
-        "sourceMap": true,                     /* Generates corresponding '.map' file. */
-        "removeComments": false,               /* Do not emit comments to output. */
-        "strict": true,                        /* Enable all strict type-checking options. */
-        "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
-        "noUnusedLocals": true,                /* Report errors on unused locals. */
-        "noUnusedParameters": true,            /* Report errors on unused parameters. */
-        "noImplicitReturns": true              /* Report error when not all code paths in function return a value. */
+        "target": "es5",
+        "allowJs": true,
+        "sourceMap": true,
+        "removeComments": false,
+        "strict": true,
+        "noImplicitAny": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noImplicitReturns": true,
+        "baseUrl": "./",
+        "paths": {
+            "@pixi.js": ["./bundles/pixi.js"],
+            "@pixi.js-legacy": ["./bundles/pixi.js-legacy"],
```

Comment:
`@pixi.js` should be `pixi.js` and `@pixi.js-legacy` should be `pixi.js-legacy.

Reply:
oops, all fixed